### ISSUE TITLE
Root evidence is being fragmented

### ISSUE THREAD

**Issue by hauck-jvsh** [sepinf-inc/IPED#2281]
The root evidence, being it an E01, or an UFDR, is being fragmented. I think that the fragmentation should not occur in the root evidence, as it will be not indexed directly.  See the example below.

![image](https://github.com/user-attachments/assets/ac30a9ae-c3bc-41b9-a04b-5a46d75e9889)

**Comment by wladimirleite** [comment]
I guess that some specific condition is causing this issue, as this doesn't happen for the E01's and UFDR's I usually process.

**Comment by wladimirleite** [comment]
@hauck-jvsh, you are passing the image file name as input ("-d"), not its parent folder, right?

**Comment by lfcnassif** [comment]
Maybe it happens when MinIO or OpenSearch tasks are enabled, since it is a common configuration used by @hauck-jvsh.

**Comment by hauck-jvsh** [comment]
Yes, I passed the E01 file int he -d option, maybe this is related to some task that is enabled. This was the actual -d option used:
"\192.168.130.10\evidencias\ArquivoMortoDeImagensJaProcessadas\ultimamilha\ultimamilha_Equipe03_OF2819707-2024_item02-D.E01\ultimamilha_Equipe03_OF2819707-2024_item02-D.E01.E01"

**Comment by lfcnassif** [comment]
@hauck-jvsh please specify the steps to reproduce, like which tasks and configuration parameters you changed, so we could reproduce and try to fix.
